protocols/redis_cache_manager.py

```
@@ -5,110 +5,112 @@
             start_time = time.time()
-            
+
             # Simulate expensive operation
             import hashlib
-            result = {
-                'computed_value': hashlib.sha256(str(time.time()).encode()).hexdigest(),
-                'computation_timestamp': datetime.utcnow().isoformat()
-            }
+
+            result = {"computed_value": hashlib.sha256(str(time.time()).encode(
+            )).hexdigest(), "computation_timestamp": datetime.utcnow().isoformat(), }
```

Ruff (0.11.9)

`datetime.datetime.utcnow()` used

(DTZ003)